doc/api/models/schemas/RewardsInfo.yml

```diff
@@ -0,0 +1,43 @@
+type: object
+description: Information about rewards configuration on the network for calculating APY
+properties:
+  epochsPerYear:
+    type: integer
+    format: int64
+    description: Number of epochs per year (based on current GS epoch configuration)
+  currentDagPrice:
+    type: integer
+    format: int64
+    description: Current DAG price (in the same format as used internally in configuration)
+  nextDagPrice:
+    $ref: './NextDagPrice.yml'
+    description: Information about the next DAG price and when it will take effect
+  totalDelegatedAmount:
+    type: integer
+    format: int64
+    description: Total amount delegated across all active delegated stakes, including accrued rewards
+  latestAverageRewardPerDag:
+    type: integer
+    format: int64
+    description: Average reward rate per DAG per epoch in datums
+  totalDagAmount:
+    type: integer
+    format: int64
+    description: Current total supply of all DAG balances in datums
+  totalRewardPerEpoch:
```

Comment:
Realizing now this might not be the most descriptive name (but it's same as in ticket.) Maybe a better name here would be lastTotalIncrementalRewards or something like that? Or we leave it as is, and communicate it as a "rate" of change. Not sure actually.